solutions/usaco-617.mdx

```diff
@@ -77,5 +77,76 @@ int main()
 ```
 
 </CPPSection>
+	
+	
+<JavaSection>
+	
+```java
+import java.util.*;
+import java.io.*;
+
+public class LoadBalancing {
+    static class Point {
+        public int x;
+        public int y;
```

Comment:
optional: add a constructor

Reply:
agree

Reply:
just delete it instead.

Reply:
why? it's still used to store the input

Reply:
aadit found that arraylists weren't necessary so he just... you know